src/lib.rs

```diff
@@ -192,6 +198,102 @@ pub enum SensorMode {
     Normal,
 }
 
+/// Oversampling settings for temperature, pressure, and humidity measurements.
+/// See sections 3.4ff of the manual for measurement flow and recommended values.
+/// The default is 1x sampling.
+#[derive(Debug, Copy, Clone)]
+pub enum Oversampling {
+    Oversampling1X,
+    Oversampling2X,
+    Oversampling4X,
+    Oversampling8X,
+    Oversampling16X,
+}
```

Comment:
The lack of comments on the enum members will trigger `deny(missing_dock)` check from the top of the file :)

Reply:
Oh, right. I didn't get that far last time, fixed now :)